src/app/leverage/constants.ts

```diff
@@ -25,3 +25,42 @@ export const leverageTokens = [...ethLeverageTokens, ...btcLeverageTokens]
 export const leverageTokenAddresses = leverageTokens
   .map((token) => token.arbitrumAddress ?? '')
   .filter((arbitrumAddress) => isAddress(arbitrumAddress))
+
+export const leverageCollateralDebt = {
+  [IndexCoopBitcoin2xIndex.arbitrumAddress!]: {
+    collateralAmountToken: '0x078f358208685046a11C85e8ad32895DED33A249', // ARB WBTC
+    collateralPriceToken: '0x2260fac5e5542a773aa44fbcfedf7c193bc2c599', // ETH WBTC
```

Comment:
what is this address for? here's the address it tracks to on arbiscan but it's not clear to me what role it plays in cost of carry: https://arbiscan.io/address/0x2260fac5e5542a773aa44fbcfedf7c193bc2c599